certified-connectors/Data247logic/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#ffffff",
```

Comment:
Unfortunately, pure white (#ffffff) isn't allowed. Using an off-white is acceptable though.